ocs_ci/ocs/resources/bucketclass.py

```
@@ -131,11 +133,26 @@ def _create_bucket_class(bucket_class_dict):
             placement_policy = None
             vector_policy = None
 
+            archive_policy = bucket_class_dict.get("archive_policy")
+
             if "vector_policy" in bucket_class_dict and interface.lower() != "oc":
                 raise RuntimeError(
                     f"vector_policy is supported only with 'oc' interface, got: {interface}"
                 )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)